IAC/Bicep/tests/spec/support/custom_matcher.sh

```diff
@@ -6,8 +6,8 @@ shellspec_matcher_include_name() {
     shellspec_matcher__match() {
         SHELLSPEC_EXPECT="$1"
         SHELLSPEC_SUBJECT=$(echo "$SHELLSPEC_SUBJECT" | jq -r '.name')
-        [ "$SHELLSPEC_EXPECT" == "$SHELLSPEC_EXPECT" ] || return 1
-        expr "$SHELLSPEC_SUBJECT" : "$SHELLSPEC_EXPECT" >/dev/null || return 1
+        [ "$SHELLSPEC_EXPECT" = "$SHELLSPEC_EXPECT" ] || return 1
```

Comment:
Can we revert these changes please? They are part of the supporting matcher code for shellspec (which isn't used.) 

We should add a story to rmove shellspec or move this implementation to the docs folder under an alternative. Especially since the tests are no longer at part with the BenchPress tests.

Reply:
fair enough, reverted changes back 👍